### PR TITLE
nodejs-conductor updated docs and Instance > DnaInstance rename

### PR DESCRIPTION
**Pull request by Connoropolous**
- [ ] I have added a summary of my changes to the changelog

### CODE REVIEW

File: doc/holochain_101/src/access_instance_info.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+### `conductor.dna_address(instanceId)` => `string`
+
+Get the address of the DNA for an instance, by passing an instance id.
+
+___
+**Name** instanceId
+
+**Type** `string`
+
+**Description** Specifies an instance by its instanceId. This instanceId should be the equivalent thing as an `instanceConfig.name` which was passed to [Config.instance](./testing_configuration.md#instances). This in turn would be equivalent to the original name given to [Config.agent](./testing_configuration.md#agents), unless you overrode it when calling [Config.instance](./testing_configuration.md#instances). See more [here](./testing_configuration.md#example-2).
```
Comment (zippy) [inline doc/holochain_101/src/access_instance_info.md]:
```suggestion
**Description** Specifies an instance by its instanceId. This instanceId should be the equivalent to an `instanceConfig.name` which was passed to [Config.instance](./testing_configuration.md#instances). This in turn would be equivalent to the original name given to [Config.agent](./testing_configuration.md#agents), unless you overrode it when calling [Config.instance](./testing_configuration.md#instances). See more [here](./testing_configuration.md#example-2).
```

File: doc/holochain_101/src/nodejs_dna_instances.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+Instantiate a `DnaInstance` based on an instanceId, and the conductor where an instance with that id is running.
+Calling this manually is not typically necessary, since the `Scenario` testing returns these natively.
+A `DnaInstance` can make calls via that Conductor into Zome functions.
+
+___
+**Name** instanceId
+
+**Type** `string`
+
+**Description** The instance id of the DnaInstance within the `conductor` passed as the second argument. If using the [Config](./testing_configuration.md) helpers, it can often be the same as "name", as taken from the optional third parameter of [Config.instance](./testing_configuration.md#instances), which itself defaults to what was given in [Config.agent](./testing_configuration.md#agents).
```
Comment (maackle) [inline doc/holochain_101/src/nodejs_dna_instances.md]:
```suggestion
**Description** The instance id of the DnaInstance as specified in the configuration of `conductor`. 
Note that when using the [Config.instance](./testing_configuration.md#instances) helper, the instance ID defaults to the agent name (as specified in [Config.agent](./testing_configuration.md#agents)) if not explicitly passed as a third argument.
```

File: doc/holochain_101/src/nodejs_dna_instances.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
+# DNA Instances
+
+`DnaInstance` is a class that is exported from `holochain-nodejs` and can be imported into your code.
```
Comment (maackle) [inline doc/holochain_101/src/nodejs_dna_instances.md]:
```suggestion
`DnaInstance` is a class that is exported from `holochain-nodejs` and can be imported into your code.
This class is used externally and instances of it are built automatically for you to use, so you typically should not have to construct a `DnaInstance` yourself.
```

I know you wrote this in the notes for the constructor but seems good to have a note right up top too.

**Comment by Connoropolous** [comment]
@maackle or @zippy can you check the failed integration test to make sure this is fine? I’m on mobile right now

**Pull request merged**